### PR TITLE
remove common file

### LINKED ISSUE #2720: Delete keyvault-common from the source tree
"Keyvault-common is abandoned"
_Originally posted by @vhvb1989 in https://github.com/Azure/azure-sdk-for-cpp/pull/2718#r685417878_

So, should we delete it from the source tree then?

### PR DESCRIPTION
**Pull request by gearama**
closes #2720 
Remove left over file in the shared folder 

# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [x] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [x] Doxygen docs
- [x] Unit tests
- [x] No unwanted commits/changes
- [x] Descriptive title/description
  - [x] PR is single purpose
  - [x] Related issue listed
- [x] Comments in source
- [x] No typos
- [x] Update changelog
- [x] Not work-in-progress
- [x] External references or docs updated
- [x] Self review of PR done
- [x] Any breaking changes?

### CODE REVIEW

**Comment by RickWinter** [comment]
Avoiding duplicating the code around in numerous files is preferred.  What is wrong with including files from a common location into each package.  We do this for other SDKs.

**Review by RickWinter — COMMENTED** [review]
Avoiding duplicating the code around in numerous files is preferred. What is wrong with including files from a common location into each package. We do this for other SDKs.

File: sdk/keyvault/azure-security-keyvault-secrets/src/secret_client.cpp [hunk 1]
```diff
@@ -9,12 +9,11 @@
 #include "azure/keyvault/secrets/secret_client.hpp"
 #include "azure/keyvault/secrets/keyvault_operations.hpp"
 #include "private/keyvault_protocol.hpp"
+#include "private/keyvault_urlScope.hpp"
```
Comment (antkmsft) [inline sdk/keyvault/azure-security-keyvault-secrets/src/secret_client.cpp:12]: Please use snake_casing and not camelCasing for file names. If there are other occurrences in our codebase (I saw one earlier), please fix them too.
```suggestion
#include "private/keyvault_url_cope.hpp"
```

**Review by antkmsft — APPROVED** [review]

**Review by vhvb1989 — CHANGES_REQUESTED** [review]
This is not good. Why are we coming back to duplicate the code?

**Comment by gearama** [comment]
We have an issue that was triaged and approved, we have some leftover dangling code that is "common", it's on folder, that contains  one file, that contains one namespace, that contains one class, that contains one in method that is 5 lines long. 
either we put all common code toghether or we completely separate things(which the issue requires that we remove ) , or we do as the issue required.

**Pull request closed without merging**